### PR TITLE
Search for packages in all GOPATHs

### PR DESCRIPTION
**Pull request by DarthSim**
There is a case: I have multiple paths in GOPATH, one for third party packages and another for my own ones. Currently party searches for packages only in the curent project's gopath, and this doesn't work in my case.
This PR solves the problem: party searches for packages in all gopaths with priority of current project's gopath.

### CODE REVIEW

**Comment by mjibson** [comment]
Sorry, could you reopen this? I forgot it was here and lost track of it.

File: main.go [hunk 1]
```diff
@@ -226,6 +231,17 @@ func update(updated map[string]bool) {
...
 }
 
+func findPackage(name string) (string, error) {
+	for _, gp := range gopath {
+		fpath := filepath.Join(gp, "src", name)
+		if _, err := os.Stat(fpath); err == nil {
+			return fpath, nil
+		}
+	}
+
```
Comment (mjibson) [inline main.go]: remove newline

**Comment by mjibson** [comment]
Tiny nit. Squash into 1 commit and I'll merge it.

**Comment by DarthSim** [comment]
@mjibson done

**Pull request merged**